.grunt/concattask.js

```diff
@@ -5,12 +5,8 @@ module.exports = {
     scripts: {
         src:
             [
-            'main-app/app/scripts/app.js',
-            'main-app/app/scripts/controllers.js',
-            'main-app/app/scripts/gamemodel.js',
-            'main-app/app/scripts/gameApi.js'
+            'main-app/app/scripts/*.js'
```

Comment:
This was probably a bad move, you need to include these in order:
1) Module(s) first
2) Providers (services, factories etc next)
3) Controllers.